devices/YarpRobotLoggerDevice/app/robots/iCubGenova09/yarp-robot-logger.xml

```diff
@@ -21,19 +22,28 @@ GNU Lesser General Public License v2.1 or any later version. -->
     </group>
   </group>
 
+  <group name="RobotCameraBridge">
+      <param name="stream_cameras">true</param>
+      <group name="Cameras">
+        <param name="rgb_cameras_list">("camera1")</param>
+        <param name="rgb_image_width">(1280)</param>
+        <param name="rgb_image_height">(720)</param>
```

Comment:
It would be better not to duplicate information

Reply:
`rgb_image_width` and `rgb_image_height` are not  anymore mandatory https://github.com/ami-iit/bipedal-locomotion-framework/pull/516/commits/487c746e8428fe22b4d0507115d170a0cfa0e424